### PR TITLE
Installation instructions update

### PR DESCRIPTION
**Pull request by ragusa**
Updating instructions for installing chi-tech.

### CODE REVIEW

**Comment by Naktakala** [comment]
These look good to me? Want to keep going on the same PR or start a new one?

**Comment by ragusa** [comment]
@Naktakala it's ready for your review. You need to look at the python 2.6 comments regarding PETSc. I feel this one is old and needs to be updated but you will know this better than I do.

File: ChiDoc/Install_ubuntu_easy.md [hunk 1]
```diff
@@ -38,21 +39,27 @@ sudo apt-get install build-essential gfortran
...
-sudo apt-get install cmake python git zlib1g-dev libx11-dev unzip
-sudo apt-get install libglu1-mesa-dev freeglut3-dev mesa-common-dev
+sudo apt-get install cmake python3 git zlib1g-dev libx11-dev unzip
 ```
+(note the use of python3)
+
 
+
+***TODO: this needs to be fixed @JANV ***
 <u>NOTE</u>: *The recommended version of PETSc requires Python 2.6+; Python 3.x, typically
```
Comment (Naktakala) [inline ChiDoc/Install_ubuntu_easy.md]: We remove this entire line. The newer PETSc versions are fine with python3 only.

**Review by Naktakala — APPROVED** [review]

**Comment by Naktakala** [comment]
@ragusa If you can just remove that one line, then I can merge.

**Comment by Naktakala** [comment]
I managed to remove the line myself. Will be merging now.

**Pull request merged**